src/components/selectionBoxLayer.ts

```diff
@@ -21,6 +21,14 @@ export module Components {
 
     constructor() {
       super();
+      /*
+       * Enable clipPath to hide _detectionEdge s and _detectionCorner s
+       * that overlap with the edge of the DragBoxLayer. This prevents the
+       * user's cursor from changing outside the DragBoxLayer, where they
+       * wouldn't be able to grab the edges or corners for resizing.
+       */
```

Comment:
I'm not a big fan of this verbosity. Delete the comment entirely.

It's also a copy+paste, this is the `selectionBoxLayer`